solidity/ecdsa/README.adoc

```diff
@@ -3,6 +3,8 @@
 
 = Keep ECDSA v2
 
+https://github.com/keep-network/keep-core/.github/workflows/solidity-test.yml[image:https://img.shields.io/github/workflow/status/keep-network/sortition-pools/Test%20Solidity?event=schedule&label=ECDSA%20contracts%20build[Solidity tests status]]
```

Comment:
The `keep-network/sortition-pools/Test` part is probably wrong.

Reply:
Right, fixed in [cfc2721](https://github.com/keep-network/keep-core/pull/3398/commits/cfc27217f3de34591b02564ee04d99e2ed7984ed).